datalad/interface/run_procedure.py

```diff
@@ -97,29 +97,94 @@ def _guess_exec(script_file):
         return u'bash "{script}" "{ds}" {args}'
     elif script_file.endswith('.py'):
         return u'python "{script}" "{ds}" {args}'
-    raise ValueError("NO IDEA")
+    raise ValueError("No idea how to execute procedure %s. Missing 'execute' permissions?", script_file)
 
 
 @build_doc
 class RunProcedure(Interface):
-    """
-    DO stuff
-    datalad.locations.dataset-procedures
-    datalad.locations.user-procedures
-    datalad.locations.system-procedures
+    """Run prepared procedures (DataLad scripts) on a dataset
+
+    *Concept*
+
+    A "procedure" is an algorithm with the purpose to process a dataset in a
+    particular way. Procedures can be useful in a wide range of scenarios,
+    like adjusting dataset configuration in a uniform fashion, populating
+    a dataset with particular content, or automating other routine tasks,
+    such as synchronizing dataset content with certain siblings.
+
+    Implementations of some procedures are shipped together with DataLad,
+    but additional procedures can be provided by 1) any DataLad extension,
+    2) any dataset, 3) a local user, or 4) a local system administrator.
+    DataLad will look for procedures in the following locations and order:
+
+    Directories identified by the configuration settings
+
+    1. 'datalad.locations.dataset-procedures'
+
+    2. 'datalad.locations.user-procedures' (on a *nix system this defaults
+       to: '$HOME/.config/datalad/procedures')
+
+    3. 'datalad.locations.system-procedures' (on a *unix system this
```

Comment:
s/*unix/*nix/, though is the broader sentence correct?  I don't think on macOS this would default to anything in "/etc/".

Reply:
No idea, but if you bring it up, probably not. Kill entirely or say Linux? Feel free to push into the PR, BTW.